webdriver/tests/new_session/headers.py

```diff
@@ -0,0 +1,6 @@
+def test_missing_first_match(new_session, add_browser_capabilities):
```

Comment:
This isn't really what this is testing. You could possibly even just use the `status` endpoint to test this since it doesn't require a session.

Reply:
oh dang, copy paste issue